tensorflow/compiler/mlir/tosa/tests/tf-to-tosa-pipeline.mlir

```diff
@@ -42,6 +42,35 @@ func.func @test_transpose_conv2d(%arg0: tensor<1x32x32x8xf32>, %arg1: tensor<1x1
 
 // -----
 
+// CHECK-LABEL: test_conv3d
+// CHECK-SAME: %[[VAL_0:.*]]: tensor<2x4x128x128x8xf32>
+// CHECK-SAME: %[[VAL_1:.*]]: tensor<2x3x3x2x4xf32>
+// CHECK: %[[VAL_2:.*]] = "tosa.const"() {value = dense<0.000000e+00> : tensor<4xf32>}
+// CHECK: %[[VAL_3:.*]] = "tosa.const"() {value = dense<[4, 0, 1, 2, 3]> : tensor<5xi32>}
```

Comment:
For all the constants, could you use CHECK-DAG instead of CHECK? (flagged in internal test which I don't know why it isn't upstream, but it is meant to not be sensitive to orderings of sequence of constants)

Reply:
Sorry for the inconvenience. Done.